beam-core/Database/Beam/Backend/SQL.hs

```diff
@@ -133,6 +133,12 @@ class (BeamBackend be, Monad m) =>
                  Nothing -> pure (Just x)
                  Just _ -> pure Nothing
 
+  -- | Run the given command and fetch the first result. The result is
+  --   'Nothing' if no results are returned.
+  --   This is not guaranteed to automatically apply LIMIT 1 to the query.
```

Comment:
`LIMIT 1` is a postgresql-ism. I would use something neutral in the description.

Reply:
Done!